docs/src/main/sphinx/release/release-376.md

```diff
@@ -0,0 +1,67 @@
+# Release 376 (6 Apr 2022)
+
+## General
+
+* Add table redirection awareness for `RENAME table` operations. ({issue}`11277`)
+* Deny adding column with comment if the connector does not support this feature. ({issue}`11486`)
+* Improve query plans and performance in the presence of inequality expressions. ({issue}`11518`)
+* Fix operator type exception that occurred when index join was used during
+  query execution. ({issue}`11758`)
+
+## Security
+
+* Add support to use two-way TLS/SSL with LDAP authentication. ({issue}`11070`)
```

Comment:
fyi @Praveen2112

Reply:
What's "two-way TLS/SSL"? Does that refer to mutual authentication (certificate validation) in SSL/TLS?

If so, maybe rephrase as:

> Add support for mutual TLS/SSL certificate validation with LDAP authentication

Reply:
Yes ... and its a pretty common term .. reworded a bit to add cert stuff